mmseg/models/losses/cross_entropy_loss.py

```diff
@@ -150,14 +150,18 @@ class CrossEntropyLoss(nn.Module):
         class_weight (list[float] | str, optional): Weight of each class. If in
             str format, read them from a file. Defaults to None.
         loss_weight (float, optional): Weight of the loss. Defaults to 1.0.
+        loss_name (str, optional): Name of the loss item. If you want this loss
+            item to be included into the backward graph, `loss_` must be the
+            prefix of the name. Defaults to 'loss_ce'.
     """
 
     def __init__(self,
                  use_sigmoid=False,
                  use_mask=False,
                  reduction='mean',
                  class_weight=None,
-                 loss_weight=1.0):
+                 loss_weight=1.0,
+                 name='loss_ce'):
```

Comment:
```suggestion
                 loss_name='loss_ce'):
```

should be consistent with docstring.